src/dataset/DatasetTree.ts

```diff
@@ -347,7 +341,7 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
                 }
             }
             if (this.mSessionNodes.length === 1) {
-                this.addSingleSession(Profiles.getInstance().getDefaultProfile(profileType));
+                await this.addSingleSession(DefaultProfileManager.getInstance().getDefaultProfile("zosmf"));
```

Comment:
@katelynienaber Can we please change this back to using the `profileType` variable rather than hard-coding it? It seems we keep the variable for USS and Jobs.

Reply:
> Hi @katelynienaber, actually I didn't try using the extension after taking that screenshot, because I assumed the error message meant other things wouldn't work. I agree with @kristinochka that having the error message is confusing if it's ok that the user has no base profile. To try and properly answer your question, I just pulled the latest version of this PR branch and tried to run it in the extension host, but was not able to as I encountered build errors. Does the extension build for you?

I am not sure what errors you are getting when trying to run it this way :) I am debugging it the same way and it is working for me...maybe this is fixed now? Or if you still get errors please send a screenshot.

Also @kristinochka @JillieBeanSim @lauren-li I removed the base profile missing error :)

Reply:
Fixed this